JSDemos/Demos/RangeSlider/Overview/Angular/description.md

```diff
@@ -0,0 +1,27 @@
+The RangeSlider component allows users to choose a range of numeric values. This demo shows how you can create a RangeSlider and configure its features.
+
+## Create a RangeSlider
+
+To create a RangeSlider, specify four properties:
```

Comment:
Is it all one needs to create a RangeSlider? Should I do anything else before specifying these properties?